tests/reward_forwarder/test_deposit_reward.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+import brownie
+
+WEEK = 86400 * 7
+
+
+def test_reward_deposit(alice, bob, charlie, child_gauge, reward_forwarder, reward_token):
+
+    reward_token._mint_for_testing(charlie, 10 ** 26, {"from": alice})
```

Comment:
Noticing some common lines like this mint for testing, set manager. Where possible it's always best to lift these into a setup fixture at the top of the test module.

```python
@pytest.fixture(scope="module", autouse=True)
def setup(alice, bob, charlie, child_gauge, reward_token):
    reward_token._mint_for_testing(charlie, 10**26, {"from": charlie})
    child_gauge.set_manager(charlie, {"from": alice})
```
This keeps unit tests as small as possible, and easy to discern what is happening.